core/src/gossiper.rs

```diff
@@ -0,0 +1,167 @@
+//! Gossiper is actor which is responsible for transaction gossiping
```

Comment:
why is it not also responsible for block gossiping?

Reply:
we already have block sync, do you think we should combine them?

Reply:
i want to keep scope of actor narrow

Reply:
let's rename it to `TransactionGossiper` then, because we have multiple forms of gossiping

Reply:
I think we can insist on difference but still use `TransactionGossiper` to highlight what we actually gossiping.